gatsby-config.js

```diff
@@ -41,7 +42,8 @@ module.exports = {
           fr: 5.3,
           pt: 5.3,
         },
-        angular: {
+        angul
+        : {
```

Comment:
@eihabkhan seems that you deleted something here? It should be "angular" not "angul". Also, the Taiwanese translation was just updated and the key "'zh-TW': 5.3," you have here is incorrect. If you wouldn't mind fixing this we'd appreciate it.

Reply:
Fixed 👌